routes/api/transaction.js

```diff
@@ -66,8 +66,13 @@ router.get(
                 currentTransactionValue["id"] = transaction.id;
                 currentTransactionValue["stockTicker"] = transaction.ticker;
                 currentTransactionValue["units"] = transaction.units;
+                currentTransactionValue["price"] = transaction.price;
                 currentTransactionValue["currentPrice"] =
                   currentTickerPrice[transaction.ticker];
+                currentTransactionValue["status"] = (
```

Comment:
let us rename this to "Profit-Loss" instead of status.

Reply:
Done.